### PR TITLE
Finished drivetrain code base(Using last year motors models)

### PR DESCRIPTION
**Pull request by kenobi1337**
Finished drivetrain code
All codes written follow the uptodate FRC command based robot best practices
All codes have concise comment
All drivetrain code based from last year robot code

### CODE REVIEW

**Comment by kenobi1337** [comment]
Y'all didn't invite me to the github team yet, so I need to do fork pull request like this

File: src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -12,4 +12,9 @@
  * <p>It is advised to statically import this class (or one of its inner classes) wherever the
  * constants are needed, to reduce verbosity.
  */
-public final class Constants {}
+public final class Constants {
+    public static final int motorFrontRightChannel = 4;
+    public static final int motorFrontLeftChannel = 1;
+    public static final int motorBackRightChannel = 3;
+    public static final int motorBackLeftChannel = 2;
```
Comment (sviridovt) [inline src/main/java/frc/robot/Constants.java:19]: NIT: Can we put constant variable names in all caps

File: src/main/java/frc/robot/subsystems/DriveTrain.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+// dependency import
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+public class DriveTrain extends SubsystemBase {
+    /** Creates a new ExampleSubsystem. */
+
+    // math constants
+    private final double delta = 0.05;
+    private final double kP = 0.015;
```
Comment (sviridovt) [inline src/main/java/frc/robot/subsystems/DriveTrain.java:22]: Same as above

**Review by sviridovt — DISMISSED** [review]

**Comment by jack60612** [comment]
we should probably change to ring controls later, but this is good for now

**Comment by jack60612** [comment]
@kenobi1337 tests failing, please fix

**Review by jack60612 — APPROVED** [review]
lgtm

**Pull request merged**